ksql-functional-tests/src/test/resources/unit_test.json

```diff
@@ -0,0 +1,169 @@
+{
+  "comments": [
+    "This is for unit test only."
+  ],
+  "tests": [
+    {
+      "name": "project and filter",
+      "statements": [
+        "CREATE STREAM TEST (ID bigint, NAME varchar, VALUE double) WITH (kafka_topic='test_topic', value_format='DELIMITED', key='ID');",
+        "CREATE STREAM S1 as SELECT name FROM test where id > 100;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 0, "value": "0,zero,0.0"},
+        {"topic": "test_topic", "key": 100, "value": "100,100,0.0"},
+        {"topic": "test_topic", "key": 101, "value": "101,101,0.0"}
+      ],
+      "outputs": [
+        {"topic": "S1", "key": 101, "value": "101"}
+      ]
+    },
+    {
+      "format": ["AVRO", "JSON"],
```

Comment:
Why have we added these additional tests to the unit_test file?

Reply:
The name is misleading, I changed it to `testing_tool_tests`.
These are different than the ones in QTT since we have more than one query in these tests.